covalent_ssh_plugin/ssh.py

```diff
@@ -45,10 +45,12 @@
     "hostname": "",
     "ssh_key_file": os.path.join(os.environ["HOME"], ".ssh/id_rsa"),
     "cache_dir": str(Path(get_config("dispatcher.cache_dir")).expanduser().resolve()),
-    "remote_cache": ".cache/covalent",
     "python_path": "python",
-    "run_local_on_ssh_fail": False,
     "conda_env": "",
+    "remote_cache": ".cache/covalent",
+    "run_local_on_ssh_fail": False,
+    "remote_workdir": ".cache/covalent/remote_workdir",
```

Comment:
Probably better to change the `remote_workdir` default to `covalent-workdir` to match with the Slurm executor.

Reply:
done!